torchgeo/datasets/reforestree.py

```diff
@@ -0,0 +1,301 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""ReforesTree dataset."""
+
+import glob
+import os
+from typing import Callable, Dict, List, Optional, Tuple
+
+import matplotlib.patches as patches
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from PIL import Image
+from torch import Tensor
+
+from .geo import VisionDataset
+from .utils import check_integrity, download_and_extract_archive, extract_archive
+
+
+class ReforesTree(VisionDataset):
+    """ReforesTree dataset.
+
+    The `ReforesTree <https://github.com/gyrrei/ReforesTree>`_
```

Comment:
You'll probably need an extra _ here for the docs to render correctly in the non_geo_datasets.csv table:
```suggestion
    The `ReforesTree <https://github.com/gyrrei/ReforesTree>`__
```